components/VotesList/VotesListItem.tsx

```diff
@@ -178,6 +178,17 @@ export function VotesListItem({
     return "transparent";
   }
 
+  function getTitleMaxWidth() {
```

Comment:
different max width algorithms depending on how much space is available in the table